CRM/Core/BAO/Dashboard.php

```diff
@@ -109,6 +109,7 @@ public static function getContactDashlets($contactID = NULL) {
       'contact_id' => $contactID,
       'is_active' => 1,
       'dashboard_id.is_active' => 1,
+      'dashboard_id.domain_id' => CRM_Core_Config::domainID(),
```

Comment:
@christianwach do we need to cope with dashboards that don't have a domain_id?

Reply:
@seamuslee001 There should always be [a `domain_id` assigned to each dashlet](https://github.com/civicrm/civicrm-core/pull/15099/files#diff-87979aa1d4055069818ccac6590029a6R394-R396) when they are created. Personally I'd consider dashlets without a `domain_id` to be orphaned.